extra/test_release_tool.py

```diff
@@ -194,38 +139,70 @@ def test_version_of(capsys):
+            os.path.join(INTEGRATION_DIR, "docker-compose.yml"),
+            os.path.join(INTEGRATION_DIR, "docker-compose.yml.bkp"),
+        )
+
+        # Manually modifying the Git version:
+        filename = os.path.join(INTEGRATION_DIR, "git-versions.yml")
+        with open(filename, "w") as fd:
+            fd.write(
+                """services:
     mender-gui:
         image: mendersoftware/gui:1.2.3-git
 """
+            )
+        run_main_assert_result(capsys, ["--version-of", "gui"], "1.2.3-git")
+        run_main_assert_result(
+            capsys,
+            ["--version-of", "gui", "--version-type", "docker"],
+            "mender-master",
+        )
+        run_main_assert_result(
+            capsys, ["--version-of", "gui", "--version-type", "git"], "1.2.3-git"
         )
-    run_main_assert_result(capsys, ["--version-of", "gui"], "1.2.3-git")
-    run_main_assert_result(
-        capsys, ["--version-of", "gui", "--version-type", "docker"], "mender-master",
-    )
-    run_main_assert_result(
-        capsys, ["--version-of", "gui", "--version-type", "git"], "1.2.3-git"
-    )
 
-    # Manually modifying the Docker version:
-    filename = os.path.join(INTEGRATION_DIR, "docker-compose.yml")
-    with open(filename, "w") as fd:
-        fd.write(
-            """services:
+        # Manually modifying the Docker version:
+        filename = os.path.join(INTEGRATION_DIR, "docker-compose.yml")
+        with open(filename, "w") as fd:
+            fd.write(
+                """services:
     mender-gui:
         image: mendersoftware/gui:4.5.6-docker
 """
+            )
+        run_main_assert_result(capsys, ["--version-of", "gui"], "1.2.3-git")
+        run_main_assert_result(
+            capsys, ["--version-of", "gui", "--version-type", "docker"], "4.5.6-docker",
         )
-    run_main_assert_result(capsys, ["--version-of", "gui"], "1.2.3-git")
-    run_main_assert_result(
-        capsys, ["--version-of", "gui", "--version-type", "docker"], "4.5.6-docker",
-    )
+        run_main_assert_result(
+            capsys, ["--version-of", "gui", "--version-type", "git"], "1.2.3-git"
+        )
+
+    finally:
+        os.rename(
+            os.path.join(INTEGRATION_DIR, "git-versions.yml.bkp"),
+            os.path.join(INTEGRATION_DIR, "git-versions.yml"),
+        )
+        os.rename(
+            os.path.join(INTEGRATION_DIR, "docker-compose.yml.bkp"),
+            os.path.join(INTEGRATION_DIR, "docker-compose.yml"),
+        )
+
+    # At the time of writing, workflows git repository has two docker images,
+    # where one has the same name as the repository (ambiguous). Make sure it
+    # still works to query this when specifically giving "docker" as the version
+    # type.
     run_main_assert_result(
-        capsys, ["--version-of", "gui", "--version-type", "git"], "1.2.3-git"
+        capsys,
+        ["--version-of", "workflows", "--version-type", "docker"],
+        "mender-master",
     )
```

Comment:
For completeness, I would add a test for the git variant:
```
    run_main_assert_result(
        capsys,
        ["--version-of", "workflows", "--version-type", "git"],
        "master",
    )
```

Reply:
Good point, added.